src/webui/www/private/views/preferences.html

```diff
@@ -2098,6 +2102,7 @@
                     document.getElementById("hideZeroFiltersCheckbox").checked = (LocalPreferences.get("hide_zero_status_filters", "false") === "true");
                     $("dblclickDownloadSelect").value = LocalPreferences.get("dblclick_download", "1");
                     $("dblclickCompleteSelect").value = LocalPreferences.get("dblclick_complete", "1");
+                    document.getElementById("confirmTorrentDeletion").checked = (LocalPreferences.get("confirm_torrent_deletion", "true") === "true");
```

Comment:
IMO the setting value should come from `pref`.
Generally, if the same option exists in GUI then it should use the same storage backend.